ns1/data_source_zone.go

```diff
@@ -0,0 +1,91 @@
+package ns1
+
+import (
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	ns1 "gopkg.in/ns1/ns1-go.v2/rest"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/dns"
+)
+
+func dataSourceZone() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"zone": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
could we get some acceptance tests for this Data Source? there's an example of doing this here: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/data_source_batch_pool_test.go

Reply:
Added.